paymentsheet/src/main/java/com/stripe/android/paymentelement/embedded/content/PaymentOptionDisplayDataHolder.kt

```diff
@@ -39,6 +45,31 @@ internal class DefaultPaymentOptionDisplayDataHolder @Inject constructor(
                         selection = selection,
                         paymentMethodMetadata = state.paymentMethodMetadata,
                     )
+                    handleImmediateRowActionCallback(
```

Comment:
I don't think this implementation will work, as it will trigger on events that aren't clicks for example.